armi/settings/fwSettings/tests/test_tightCouplingSettings.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2019 TerraPower, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""XS Settings tests"""
```

Comment:
This should be changed (looks like a copy paste from test_crossSectionSettings.py that was just missed).

Reply:
Yeah, please update it

Reply:
https://github.com/terrapower/armi/pull/1033/commits/e40104bd31aa10a5967a161ffd27be3e8439fb79